inspirehep/modules/records/jsonschemas/records/institutions.json

```diff
@@ -131,20 +131,30 @@
             "type": "array",
             "uniqueItems": true
         },
-        "obsolete_ICN": {
-            "description": "ICN of obsolete inst for which this inst should be used instead",
-            "title": "Obsolete ICN",
-            "type": "string"
-        },
-        "obsolete_record": {
-            "$ref": "elements/json_reference.json",
-            "description": "record URI of obsolete inst for which this inst should be used instead",
-            "title": "Obsolete record URI"
-        },
-        "old_ICN": {
-            "description": "HEP affiliation (spires name)",
-            "title": "Old SPIRES control number",
-            "type": "string"
+        "superseded_institutions": {
+            "items": {
+                "additionalProperties": false,
+                "properties": {
+                    "curated_relation": {
+                        "description": "Did a cataloguer proof-checked the recid?",
+                        "title": "The affiliation is curated?",
```

Comment:
Yep! So we use institute and institutions interchangeably. We should stick with one, but this will be the subject of another PR.